idl/internal/docstring_test.go

```diff
@@ -0,0 +1,118 @@
+package internal
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCleanupDocstring(t *testing.T) {
+	tests := []struct {
+		give []string
+		want []string
+	}{
+		{},
+		{
+			give: []string{"/** foo bar */"},
+			want: []string{"foo bar"},
+		},
+		{
+			give: []string{
+				"/**",
+				" * foo",
+				" *   bar",
+				" * baz",
+				" */",
+			},
+			want: []string{
+				"foo",
+				"  bar",
+				"baz",
+			},
+		},
+		{
+			give: []string{
+				"	/**",
+				"	 * foo",
+				"	 * 	bar",
+				"	 * baz",
+				"	 */",
+			},
+			want: []string{
+				"foo",
+				"	bar",
+				"baz",
+			},
+		},
+		{
+			give: []string{
+				"/**",
+				"	 * hello",
+				"	 * world",
+				"	 */",
+			},
+			want: []string{
+				"hello",
+				"world",
+			},
+		},
+		{
+			give: []string{
+				"/**",
+				"	 * hello",
+				"	 *",
+				"	 * world",
+				"	 */",
+			},
+			want: []string{
+				"hello",
+				"",
+				"world",
+			},
+		},
+		{
+			give: []string{
+				"/**",
+				"	 * hello",
+				"",
+				"	 * world",
+				"	 */",
+			},
+			want: []string{
+				"hello",
+				"",
+				"world",
+			},
+		},
+		{
+			give: []string{
+				"	/**",
+				"	 *foo",
+				"	 *	bar",
+				"	 *baz",
+				"	 */",
+			},
+			want: []string{
+				"foo",
+				"	bar",
+				"baz",
+			},
+		},
+		{
+			give: []string{
+				"/**",
+				"	    * foo does stuff",
+				"	    */",
+			},
+			want: []string{"foo does stuff"},
+		},
+	}
```

Comment:
can you add a test case where the lines inside are missing the ` * `? e.g.,
```
give : []string{
  "/**",
  "no prefix",
  "for lines",
  "*/",
},
```

and maybe another case where we have a mix of lines with the prefix and without